Sorting.cpp

```diff
@@ -0,0 +1,63 @@
+#include "Sorting.h"
+
+
+
+Sorting::Sorting()
+{
+}
+
+
+Sorting::~Sorting()
+{
+
+}
+
+double * Sorting::selectionSort(double * arr, int length)
```

Comment:
nie podoba mi się, że musisz podawać jako argument tablicę oraz jej długość, tak jakby nie można było zmierzyć długości wewnątrz funckji(?) czy to jakiś element optymalizacji wydajności? Dziwne.